### PR TITLE
📝Slack test : update Readme with new Oauth Configuration

### PR DESCRIPTION
**Pull request by zbendhiba**

### CODE REVIEW

**Comment by zbendhiba** [comment]
Oh I forgot something to add for this one

**Review by oscerd — APPROVED** [review]

**Comment by zbendhiba** [comment]
I've changed the channel name, as I had some issues with French name of the #3531 general channel

File: integration-tests/slack/README.adoc [hunk 1]
```diff
@@ -4,7 +4,32 @@ By default the Slack integration tests use WireMock to stub the API interactions
 
 To run the `camel-quarkus-Slack` integration tests against the real API, you must first create a Slack account and workspace https://slack.com/get-started#/create.
 
-Then configure Slack custom configurations for 'Incoming Webhooks' & the 'Slack API Tester' on your workspace.
+Create a channel named `test-channel`.
+
+You’ll need to create a Slack app and use it on your workspace.
+
+Use the 'User OAuth Token' as token for the producer abd consumer endpoints. Set the corresponding scopes for Bot Token:
```
Comment (jamesnetherton) [inline integration-tests/slack/README.adoc:11]:
```suggestion
Use the 'User OAuth Token' as token for the producer and consumer endpoints. Set the corresponding scopes for Bot Token:
```

**Review by jamesnetherton — COMMENTED** [review]
Small typo nitpick. Otherwise LGTM.

**Comment by zbendhiba** [comment]
> Small typo nitpick. Otherwise LGTM.

Thanks !!

**Review by jamesnetherton — APPROVED** [review]

**Review by aldettinger — APPROVED** [review]

**Pull request merged**